buildSrc/src/main/kotlin/otel.japicmp-conventions.gradle.kts

```diff
@@ -21,6 +21,10 @@ plugins {
  * The latest *released* version of the project. Evaluated lazily so the work is only done if necessary.
  */
 val latestReleasedVersion: String by lazy {
+  // TODO(anuraaga): Restore code after 1.10.0 is released
+  val moduleVersion = "1.9.1"
```

Comment:
@jkwatson This was the only idea that came to mind to merge this PR (which is otherwise comparing against RC1 and flagging the breaking change)